docs/SUBWORKFLOWS.md

```diff
@@ -0,0 +1,147 @@
+# Adding a new subworkflow to nf-scil
+
+## Generate the template
+
+First verify you are located at the root of this repository (not in `subworkflows`), then run the following interactive command :
+
+```
+nf-core subworkflows create
+```
+
+It will ask you to give a name for your subworkflow and an author name (use your github username).
+Alternatively, you can use the following command to supply directly those informations :
+
+```
+nf-core subworkflows create <name> --author <author>
+```
+
+## Generate the template
+
+### Edit `./subworkflows/nf-scil/<name_of_your_workflow>/main.nf`
+
+You don’t have the choice to generate an empty template when you generate a subworflow, so the template is based on nf-core.
+
+- Remove the different comment lines.
+- Include your modules into your subworkflows. Remove the modules `{ SAMTOOLS_SORT}` and `{ SAMTOOLS_INDEX }` then includes yours with the good pathway:
+
+```
+include { <MODULES>	} from '../../../modules/nf-scil/<category>/<tool>/main'
+```
+
+> [!NOTE]
+> You can also include other subworkflows :
+>
+> ```
+> include { <SUBWORKFLOW> } from '../<subworkflow>/main'
+> ```
+
+#### Define your Workflow inputs.
+
+A workflow can declare one or more input channels using the `take` keyword.
+Multiple inputs must be specified on separate lines:
```

Comment:
I would add some details for optional input channels. I made some suggestions but feel free to modify them. Also, I can't suggest modifications in the code snippet, but I would add the `, optional` to the `channel_data2` or add a third channel. 
```suggestion
A workflow can declare one or more input channels using the `take` keyword. If some of your input channels are optional, add an optional tag after the channel specification.
Multiple inputs must be specified on separate lines:
```